### DISCUSSION TITLE
recreate the same environment on my local system

### DISCUSSION THREAD

**Discussion by sambitraze** [directus/directus#10749] [Archive]
can anyone write the full step of migration?
i have the original files on the server only ... i have pushed the YAML file and .env file to GitHub now how should i proceed to work on the project locally before updating them to the server?

Till now i pulled the repo and then created an empty database with the same name as that of the server and i installed pg and directus now what to do next?

i am trying to run now `npx directus schema apply snapshot.yaml` but it shows me that i should run `directus bootstrap` first....so i ran the bootstrap command now when i am trying to run `npx directus schema apply snapshot.yaml` command but it is showing the following error.

18:27:22 🚨 Failed to create relation "directus_collections.group"
18:27:22 🚨 Field "group" in collection "directus_collections" already has an associated relationship 
Error: Field "group" in collection "directus_collections" already has an associated relationship

**Comment by benhaynes** [comment]
For my local dev environment (git clone), I use this to run migrations:

```
npm run cli bootstrap 
```

But I agree, we need to make sure all of this is in the Docs.

**Reply by sambitraze** [reply]
The cli scripts are only available when we fork the directus repo for contribution purposes.. what i want is like i have created a project and set up entire models and schema in the ec2 server but i know want to change a few things and run in my local system before applying to the main server.

**Reply by sambitraze** [reply]
It would be a great help if you and your team can improve the migration guide for migration of projects over github. : )

**Comment by rijkvanzanten** [comment]
@sambitraze That particular error looks like a bug we've seen before IIRC. Are you on the latest version of Directus in both environments? 🤔

**Reply by sambitraze** [reply]
no i am on 9.2.2 on both versions ... shall i update both of them?

**Reply by rijkvanzanten** [reply]
I vaguely remember seeing an issue flash by recently that mentions that directus_collections.group field, but I can't find it now. Always recommended to be on the latest version though

**Reply by sambitraze** [reply]
thanks updating on both side helped
